ProxyHandler.class.php

```diff
@@ -296,13 +314,11 @@ protected function readResponse(&$cu, $body)
                 $this->_removeHeader('Pragma');
             }
             $info = $this->getCurlInfo();
-            if (isset($info['content_type'])) {
+            if (isset($info['content_type']) && !isset($this->_contentType)) {
```

Comment:
Make this a local function variable `$contentType` and work with that to make this more readable